### PR TITLE
added docker cheatsheet

### PR DESCRIPTION
**Pull request by ayeolakenny**
### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [x] Search for duplicates.
- [x] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: CheatSheets/docker-cheatsheet.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+## Table of Contents
+
+- [Conda CheatSheet for Developers](#conda-cheatsheet-for-developers)
```
Comment (crescentpartha) [inline CheatSheets/docker-cheatsheet.md:3]:
```suggestion
- [Docker CheatSheet for Developers](#docker-cheatsheet-for-developers)
```

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Please, `modify the link` and your `docker-cheatsheet.md` add to the `README.md` file.

File: CheatSheets/docker-cheatsheet.md (hunk 1 above)
Comment (ayeolakenny) [reply CheatSheets/docker-cheatsheet.md]: Fixed the title issue and added it to the readme

**Review by crescentpartha — APPROVED** [review]
Approved your contributions.

**Pull request merged**